Tests/MoyaSugarTests/GitHubAPI.swift

```diff
@@ -79,8 +78,28 @@ enum GitHubAPI: SugarTargetType, Then {
   var sampleData: Data {
     return Data()
   }
+}
 
-  var task: Task {
-    return .request
+// Extension for Testing
+extension GitHubAPI {
```

Comment:
We don't need this anymore.

Reply:
We need it to test `func testParams()` from `SugarTargetTypeTests.swift`.

Reply:
We probably can test the enum case of `task` instead.